### DISCUSSION TITLE
Unable to fix Microsoft To Do list to Start Menu

### DISCUSSION THREAD

**Discussion by ShiranuiAegis** [valinet/ExplorerPatcher#2991] [Q&A]
I can't fix the list in Microsoft To Do to the Start Menu when I use Windows 10 Start Menu.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Reply by ShiranuiAegis** [reply]
My Windows version is Windows 11 23H2 22631.3269
explorer patcher version is 22621.3269.64.1
![屏幕截图 2024-03-13 222236](https://github.com/valinet/ExplorerPatcher/assets/89692781/ff218fdf-c375-407d-80f6-cd3fe2f0558d)
![屏幕截图 2024-03-13 222257](https://github.com/valinet/ExplorerPatcher/assets/89692781/a147b3b8-209f-4a66-be3f-ecfd8068c83f)
![屏幕截图 2024-03-13 222312](https://github.com/valinet/ExplorerPatcher/assets/89692781/a6a40f1d-779a-4569-9641-8a1e6158bc3a)
![屏幕截图 2024-03-13 222329](https://github.com/valinet/ExplorerPatcher/assets/89692781/b7bf472e-de5c-4c24-8025-8b9e72850249)

**Comment by pyrates999** [comment]
In EP properties application, click on file explorer, untick register as shell extension and click restart file explorer in the bottom left corner.

**Reply by ShiranuiAegis** [reply]
Unfortunately, it's still not possible to fix third-party tiles to the start menu

**Reply by pyrates999** [reply]
Please wait for a developer to respond.
